server/commands/documentCollaborativeUpdater.ts

```diff
@@ -36,48 +41,41 @@ export default async function documentCollaborativeUpdater({
     const node = Node.fromJSON(schema, yDocToProsemirrorJSON(ydoc, "default"));
     const text = serializer.serialize(node, undefined);
     const isUnchanged = document.text === text;
-    const hasMultiplayerState = !!document.state;
+    const lastModifiedById = userId ?? document.lastModifiedById;
 
-    if (isUnchanged && hasMultiplayerState) {
```

Comment:
This check is no longer needed as this code will never be called unless there is `state` in the database, it's created if missing as part of `onLoadDocument` hook